include/crypto/xform.h

```diff
@@ -0,0 +1,111 @@
+/****************************************************************************
+ * include/crypto/xform.h
+ * $OpenBSD: xform.h,v 1.32 2021/10/22 12:30:53 bluhm Exp $
```

Comment:
Should we have this guarded by `CONFIG_ALLOW_BSD_COMPONENTS`?

Reply:
yes，it use CONFIG_ALLOW_BSD_COMPONENTS select now